### PR TITLE
Automatically clear the cache after running the Behat test suite

### PR DESCRIPTION
**Pull request by dunglas**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

It will make contributing easier.

### CODE REVIEW

File: features/bootstrap/CacheContext.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+     * @AfterSuite
+     */
+    public static function cleanCache()
+    {
+        $fs = new Filesystem();
+        $cacheDir = __DIR__.'/../../tests/Fixtures/app/cache';
+
+        if ($fs->exists($cacheDir)) {
+            $fs->remove($cacheDir);
+            $fs->mkdir($cacheDir);
```
Comment (teohhanhui) [inline features/bootstrap/CacheContext.php]: We should just delete everything inside `$cacheDir`, not delete the directory itself.

Comment (soyuka) [inline features/bootstrap/CacheContext.php]: no

**Comment by soyuka** [comment]
Can we merge this?

**Comment by dunglas** [comment]
No the AppVeyor error looks related.

**Comment by soyuka** [comment]
Did you try to restart the appveyor build? Anyway, we don't really care that appveyor can remove the cache, however this is really useful when running tests locally. Therefore, we might just catch the exception silently, wdyt?

**Pull request closed without merging**